querybook/server/models/board.py

```diff
@@ -78,19 +65,50 @@ class BoardItem(CRUDMixin, Base):
     table_id = sql.Column(
         sql.Integer, sql.ForeignKey("data_table.id", ondelete="CASCADE"), nullable=True
     )
-    board_id = sql.Column(
-        sql.Integer, sql.ForeignKey("board.id", ondelete="CASCADE"), nullable=False
+    board_id = sql.Column(sql.Integer, sql.ForeignKey("board.id"), nullable=True)
+
+    parent_board_id = sql.Column(
```

Comment:
need to create alembic migration to rename board_id -> parent_board_id and add a new column called board_id